src/main/java/us/tastybento/bskyblock/database/objects/Island.java

```diff
@@ -104,13 +97,13 @@ public void setUniqueId(String uniqueId) {
     public Island() {}
     
     public Island(Location location, UUID owner, int protectionRange) {
-        this.members.add(owner);
         this.owner = owner;
+        this.members.put(owner, RanksManager.OWNER_RANK);
         this.createdDate = System.currentTimeMillis();
         this.updatedDate = System.currentTimeMillis();
         this.world = location.getWorld();
         this.center = location;
-        this.range = BSkyBlock.getInstance().getSettings().getIslandProtectionRange();
+        this.range = BSkyBlock.getInstance().getSettings().getIslandDistance();
```

Comment:
Bug here. range should probably be renamed to distance.

Reply:
Is it useful to store the island distance in every island?
IMO, it's not.